internal-registry/che-incubator/command-line-terminal-dev/nightly/meta.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+publisher: che-incubator
+name: command-line-terminal
+version: nightly
+type: Che Editor
+displayName: Command Line Terminal
+title: Command Line Terminal
+description: Command Line Terminal provides the ability to start a terminal inside
+  the OpenShift Console.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-machine-exec/
+firstPublicationDate: "2020-05-13"
```

Comment:
I know it changes nothing but it's copied/pasted date )

Reply:
Good catch :)